oeps/oep-0002-bp-repo-metadata.rst

```diff
@@ -60,34 +69,86 @@ Each repo will include a file ``openedx.yaml``, with the following keys:
     -  If the ``state`` is ``False`` a ``reason`` value must be included that explains why.
     -  If the ``state`` is ``True`` or ``applicable`` is ``False`` the ``reason`` can optionally provide more information. The ``reason`` value will be displayed in reporting tools. If an OEP isn't listed in the ``oeps`` dictionary, then it is assumed to be ``False``, unless the reporting tool can auto-detect accordance.
 
-``track-pulls``: obsolete
-    This key is obsolete, please remove if found.
-
-``openedx-release``: defined by :doc:`OEP-10 <oep-0010-proc-openedx-releases>`.
+``openedx-release``: defined by :doc:`OEP-10 <oep-0010-proc-openedx-releases>` (optional)
     Define this key if your repo is an application or IDA that is part of Open edX releases.  Omit this key if your repo is a library, or is not part of Open edX releases.  See :doc:`OEP-10 <oep-0010-proc-openedx-releases>` for details.
 
 ``archived``: boolean (optional)
     If ``True``, this specifies that this repository is archived and no longer maintained by edX.
 
+Obsolete Keys
+*************
+
+``track-pulls``:
+    This key is obsolete, please remove if found.
+
+``supporting_teams``: list of strings
+    This is an obsolete key that was used to specify Github handles for teams supporting the owner of the repository. This is now deprecated in favor of the updated ``owner`` dictionary. As a reasonable migration of this key, set ``owner.type`` to *team* and ``owner.value`` to the first value in ``supporting_teams``.
```

Comment:
NIT: Can we also add `Then remove this key.` to the end of this paragraph.

Reply:
Done.